dotcom-rendering/src/components/StickyLiveblogAsk.importable.tsx

```diff
@@ -0,0 +1,309 @@
+								fill="white"
+							/>
+							<path
+								d="M96.1427 44.7687C96.1427 45.8289 96.1581 46.8901 96.1369 47.9503C96.1225 48.6765 96.0897 48.6745 95.3952 48.5934C93.3779 48.3588 91.3607 48.2912 89.3618 48.7267C88.4871 48.9169 88.5026 48.9497 88.4534 49.8429C88.2107 54.2103 87.9621 58.5776 87.7194 62.944C87.6905 63.4577 87.7068 63.9753 87.6567 64.486C87.6134 64.935 87.8157 65.1262 88.2203 65.2199C88.9698 65.3927 89.7135 65.5897 90.4591 65.779C90.645 65.8263 90.8425 65.862 90.9234 66.0754C91.065 66.4491 90.8598 66.7233 90.4225 66.6992C87.8908 66.5601 85.3602 66.4095 82.8295 66.2656C80.9077 66.1565 78.9849 66.0542 77.0621 65.947C76.801 65.9325 76.5419 65.9257 76.5217 65.5511C76.5043 65.218 76.6845 65.0876 76.9648 65.0297C77.5457 64.909 78.1246 64.7757 78.7103 64.6811C79.0735 64.6222 79.2209 64.458 79.2411 64.0824C79.5966 57.4826 79.9607 50.8838 80.3364 44.285C80.3615 43.8456 80.2295 43.61 79.8172 43.4468C79.192 43.1987 78.5938 42.882 77.9801 42.6048C77.6815 42.4697 77.6709 42.2273 77.6998 41.9589C77.7287 41.6846 77.9185 41.647 78.1458 41.6277C81.1822 41.3621 84.2177 41.086 87.2541 40.8233C88.7964 40.6901 88.8031 40.7007 88.7203 42.2476C88.6268 43.9982 88.518 45.7478 88.4245 47.4984C88.4158 47.6539 88.2819 47.9156 88.517 47.9494C88.6943 47.9754 88.7299 47.7031 88.7809 47.539C89.2597 45.9989 89.8724 44.5264 90.7972 43.19C92.0168 41.4268 94.0513 40.5462 95.9654 40.974C96.3325 41.056 96.4808 41.2086 96.4577 41.591C96.3922 42.6502 96.3527 43.7114 96.3036 44.7716C96.2506 44.7716 96.1976 44.7716 96.1446 44.7716L96.1427 44.7687Z"
+								fill="white"
+							/>
+							<path
+								d="M144.55 99.8873C141.883 99.9018 139.87 97.8982 139.847 95.2071C139.825 92.543 141.803 90.5655 144.496 90.5587C147.172 90.5529 149.237 92.5179 149.245 95.0796C149.255 97.8151 147.245 99.8718 144.549 99.8863L144.55 99.8873Z"
+								fill="white"
+							/>
+						</svg>
+					</div>
+					<div css={textBlock}>
+						<div css={bodySection}>
+							The Guardian's expert news coverage is funded by
```

Comment:
```suggestion
							The Guardian’s expert news coverage is funded by
```

Also, this SVG is massive. Here it is more than three times smoler:
![Minisale](https://github.com/guardian/dotcom-rendering/assets/6032869/ba772fcf-9e14-4262-9691-0028e68cad47)
(still, like the original, ever so slightly cut off from the bottom)

I’m no designer, but the text wrap looks off in the screenshot too. Maybe [this CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/text-wrap#balance) would help? Or, if it doesn’t change between breakpoints, you could balance it manually (I would at least move `Will` onto a new line).